### PR TITLE
*: add parser and definition for all unsupported builtin funcs (part 7) 

### PR DESCRIPTION
**Pull request by XuHuaiyu**
This commit serves for the following builtin funcs:

ASYMMETRIC_DECRYPT, ASYMMETRIC_DERIVE, ASYMMETRIC_ENCRYPT,
ASYMMETRIC_SIGN, ASYMMETRIC_VERIFY, COMPRESS, CREATE_ASYMMETRIC_PRIV_KEY,
CREATE_ASYMMETRIC_PUB_KEY, CREATE_DH_PARAMETERS, CREATE_DIGEST,
DECODE, DES_DECRYPT, DES_ENCRYPT, ENCODE, ENCRYPT, MD5, OLD_PASSWORD,
PASSWORD, RANDOM_BYTES, SHA1, SHA, SHA2, UNCOMPRESS, UNCOMPRESSED_LENGTH,
VALIDATE_PASSWORD_STRENGTH

PTAL @shenli @coocood @hanfei1991 @zimulala @tiancaiamao

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -349,6 +349,30 @@ import (
 	conv		"CONV"
 	bitXor		"BIT_XOR"
 	crc32		"CRC32"
+	asymmetricDecrypt	"ASYMMETRIC_DECRYPT"
```
Comment (hanfei1991) [inline parser/parser.y]: adjust the layout

**Comment by shenli** [comment]
LGTM
Please resolve the conflicts.

**Comment by coocood** [comment]
LGTM
Well done!

**Comment by XuHuaiyu** [comment]
@hanfei1991 PTAL

File: parser/parser.y [hunk 2]
```diff
@@ -221,163 +221,163 @@ import (
...
+	bin				"BIN"
+	ceil				"CEIL"
+	ceiling				"CEILING"
+	coalesce			"COALESCE"
+	coercibility			"COERCIBILITY"
+	concat				"CONCAT"
+	concatWs			"CONCAT_WS"
+	connectionID			"CONNECTION_ID"
+	convertTz			"CONVERT_TZ"
+	curTime				"CUR_TIME"
```
Comment (zimulala) [inline parser/parser.y]: Format alignment.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**